apps/admin-portal/src/components/applications/components/url-input-component.tsx

```diff
@@ -54,14 +55,16 @@ export const URLInputComponent: FunctionComponent<URLInputComponentInterface> =
         value,
         hint,
         required,
-        disabled
+        disabled,
+        hideComponent
     } = props;
 
     const [changeUrl, setChangeUrl] = useState("");
     const [predictValue, setPredictValue] = useState([]);
     const [validURL, setValidURL] = useState(true);
     const [duplicateURL, setDuplicateURL] = useState(false);
     const [keepFocus, setKeepFocus] = useState(false);
+    const [hideEntireComponent, setHideEntireComponent] = useState(false);
```

Comment:
Shall we add types to these state variables. Check other places as well.

```suggestion
    const [hideEntireComponent, setHideEntireComponent] = useState<boolean>(false);
```